doc/ondemand_design.md

```diff
@@ -737,12 +737,12 @@ At this time we recommend the On Demand API in the following cases:
 Good applications for the On Demand API might be:
 
 * You are working from pre-existing large JSON files that have been vetted. You expect them to be well formed according to a known JSON dialect and to have a consistent layout. For example, you might be doing biomedical research or machine learning on top of static data dumps in JSON.
-* You have a closed system on predetermined hardware. Both the generation and the consumption of JSON data is within your system. Your team controls both the software that produces the JSON and the software the parses it, your team knows and control the hardware. Thus you can fully test your system.
+* Both the generation and the consumption of JSON data is within your system. Your team controls both the software that produces the JSON and the software the parses it, your team knows and control the hardware. Thus you can fully test your system.
 * You are working with stable JSON APIs which have a consistent layout and JSON dialect.
 
-## Checking Your CPU Selection
+## Checking Your CPU Selection (x64 systems)
 
-Given that the On Demand API does not offer runtime dispatching, your code is compiled against a specific CPU target. You should
+Given that the On Demand API offer limited runtime dispatching, your code is compiled against a specific CPU target. You should
```

Comment:
offer -> offers

Reply:
Given that I took your advice and removed the -march=native discussion, then this gets deleted.